### PR TITLE
feat(ucs): add new resource fleet

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/ucs' TESTARGS='-run TestAccFleet_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ucs -v -run TestAccFleet_basic -timeout 360m -parallel 4
=== RUN   TestAccFleet_basic
=== PAUSE TestAccFleet_basic
=== CONT  TestAccFleet_basic
--- PASS: TestAccFleet_basic (29.34s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ucs       29.419s
```

### CODE REVIEW

File: docs/resources/ucs_fleet.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+variable "policy_id_2" {}
+variable "policy_id_3" {}
+
+resource "huaweicloud_ucs_fleet" "test" {
+  name        = "fleet_1"
+  description = "created by terraform"
+
+  permissions {
+    namespaces = ["*"]
+    policy_ids = ["var.policy_id_1"]
```
Comment (houpeng80) [inline docs/resources/ucs_fleet.md]:
```suggestion
    policy_ids = [var.policy_id_1]
```

File: huaweicloud/services/acceptance/ucs/resource_huaweicloud_ucs_fleet_test.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    policy_ids = [huaweicloud_ucs_policy.test1.id, huaweicloud_ucs_policy.test2.id]
+  }
+}
+`, name)
+}
+
+func testFleet_update_2(name string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_ucs_fleet" "test" {
+  name        = "%[1]s"
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/ucs/resource_huaweicloud_ucs_fleet_test.go]:
```suggestion
  name = "%[1]s"
```

File: docs/resources/ucs_fleet.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+  permissions {
+    namespaces = ["default", "kube-system"]
+    policy_ids = ["var.policy_id_2","var.policy_id_3"]
```
Comment (houpeng80) [inline docs/resources/ucs_fleet.md]:
```suggestion
    policy_ids = [var.policy_id_2,var.policy_id_3]
```

Comment (Jason-Zhang9309) [reply docs/resources/ucs_fleet.md]: fixed

File: huaweicloud/services/acceptance/ucs/resource_huaweicloud_ucs_fleet_test.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/acceptance/ucs/resource_huaweicloud_ucs_fleet_test.go]: fixed

File: docs/resources/ucs_fleet.md (hunk 1 above)
Comment (Jason-Zhang9309) [reply docs/resources/ucs_fleet.md]: fixed

**Pull request merged**